src/ScreenEvaluation.cpp

```diff
@@ -729,6 +729,13 @@ bool ScreenEvaluation::Input( const InputEventPlus &input )
 	if( IsTransitioning() )
 		return false;
 
+	if (input.MenuI == GAME_BUTTON_RESTART && input.type == IET_RELEASE &&
+		GAMESTATE->IsEventMode() && !GAMESTATE->IsCourseMode())
```

Comment:
Both for gameplay and evaluation, I wonder if instead of checking eventmode/coursemode, we should make it a theme metric so that themers can control whether or not they want to restart or not.

Reply:
Sounds complicated for me. I just did small baby steps here. One logical change at once. If something like this should be implemented it should be implemented in separate PR I think. So I left it same.